torchci/pages/testing_overhead.tsx

```diff
@@ -73,26 +86,29 @@ function GenerateOncallTestingOverheadLeaderboard({
               filterable: false,
             }, 
             {
-                field: "percentage",
+                field: "percentage_of_time",
                 headerName: "% of workflow time",
                 flex: 1,
                 valueFormatter: (params: GridValueFormatterParams<number>) =>
                   params.value + "%",
                 filterable: false,
               }, 
               {
-                field: "oncall",
-                headerName: "Oncall",
+                field: "estimated_price_per_run_in_dollars",
```

Comment:
This is ok as the first step, but I think we should figure out a way to show the price = price per run x the number of workflow run.  Mainly to point out the total cost associated with each team / oncall.  Seeing a price per run of $19.23 is only half the picture I think

Reply:
And showing a bigger bill also has the benefit of nudging people to address it quicker I guess ;)